### PR TITLE
Implement function to add an org to an existing consortium

### PR DESCRIPTION
**Pull request by wlahti**
#### Type of change

- New feature

#### Description

Implement a function to add an organization to an existing consortium

#### Related issues

[FAB-17510](https://jira.hyperledger.org/browse/FAB-17510)

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -559,3 +571,43 @@ func concatenateBytes(data ...[]byte) []byte {
 	}
 	return bytes
 }
+
+// AddOrgToConsortium adds an org definition to a named consortium in a given
+// channel configuration.
+func AddOrgToConsortium(org *Organization, consortium, channelID string, config *cb.Config, mspConfig *mb.MSPConfig) (*cb.ConfigUpdate, error) {
+	if org == nil {
+		return nil, errors.New("organization is empty")
```
Comment (C0rWin) [inline pkg/config/config.go:691]: nit: "organization is nil" or "organization was not provided"

File: pkg/config/config.go [hunk 2]
```diff
@@ -559,3 +571,43 @@ func concatenateBytes(data ...[]byte) []byte {
...
+	}
+	if consortium == "" {
+		return nil, errors.New("consortium is empty")
+	}
+
```
Comment (C0rWin) [inline pkg/config/config.go:696]: a nit: but IMO worth adding checks for rest of the parameters? Briefly inspecting the code flow nil config or mspConfig potentially could lead to a panic. (I know this cannot really happen, but would add this for sanity check).

**Comment by caod123** [comment]
@wlahti feel free to reset this back to master for the dependent commit in the stack, it's been merged now

**Pull request merged**